src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -219,4 +219,9 @@ public <T extends ScriptEvent> Set<T> getEvents(Class<T> type) {
 		);
 	}
 
+	@Override
+	public String toString() {
+		return this.getConfig().getFileName();
```

Comment:
This should be the path string, or atleast the furthest minimum the /scripts/ directory parent as there can be different directories pertaining the same script file name.
```suggestion
		Config config = this.getConfig();
		return Optional.ofNullable(config::getPath).orElse(config::getFileName);
```

Reply:
Would need Optional import if using suggestion.